src/chapters/interfaces-and-polymorphism/testing-interfaces.rst

```diff
@@ -0,0 +1,59 @@
+Unit Testing and Interfaces
+===========================
+
+After all that we have learned about interfaces, perhaps you are wondering, *how do I write my unit tests with interfaces?*
+
+.. TODO: add testing inheritance link when available
+
+The best practices to testing interfaces are very similar to those of testing inheritance. You want to focus on testing the contract 
+that the interface is supposed to be upholding as opposed to the interface itself.
+
+.. admonition:: Example
+
+   We have an ``IFeedable`` interface and a ``HouseCat`` class.
+
+   .. sourcecode:: c#
+      :linenos: 
+
+      public interface IFeedable
+      {
+         string Eat()
+         {
+            return "the feedable is eating";
+         }
+      }
+
+      public class HouseCat : IFeedable
+      {
+         public string Name { get; set; }
+         public HouseCat(string name)
+         {
+            Name = name;
+         }
+      }
+
+      [TestClass]
+      public class CatTests
+      {
+         [TestMethod]
+         public void TestHouseCatImplementsEatMethod()
+         {
+            IFeedable test_cat = new HouseCat("test");
+            Assert.AreEqual("the feedable is eating", test_cat.Eat());
+         }
+      }
```

Comment:
Could these code blocks be broken up with more explanation between? Statements like, "If our code works, then ``Assert.AreEqual("the feedable is eating", test_cat.Eat());`` evaluates to true." might be really helpful to reinforce the concepts.